bindings/rust/s2n-tls/src/error.rs

```diff
@@ -191,6 +199,7 @@ impl Error {
     pub fn debug(&self) -> Option<&'static str> {
         match self.0 {
             Context::InvalidInput => None,
+            Context::CallbackExection => None,
```

Comment:
I am inclined to keep this because otherwise the next person wont be warned of the missing case and its not verbose enough to be annoying yet.

Also `ErrorSource` is an non_exhaustive enum so we could potentially add a different variant.